config/rbac/service_account.yaml

```diff
@@ -3,3 +3,5 @@ kind: ServiceAccount
 metadata:
   name: controller-manager
   namespace: system
+imagePullSecrets:
+  - name: "${IMAGE_PULL_SECRETS}"
```

Comment:
This piece should not be in this context as it is specific to OpenShift Templates and should be handled (patched) by `config/templated`.